model/GlobalConfig.php

```diff
@@ -107,9 +122,18 @@ private function initializeConfig()
      */
     private function parseConfig($filename)
     {
+        if (str_starts_with($filename, self::HTTP_URL_PREFIX) || str_starts_with($filename, self::HTTPS_URL_PREFIX)) {
```

Comment:
Maybe we could modify `parseConfig` to pass a flag telling it whether it is a local config, or a remote config? This way we wouldn't have to check if it starts with `http` or `https` here and in the caller function :point_up: